adopterAPI.json

```diff
@@ -0,0 +1,457 @@
+          "ApplicationsEndpoints"
+        ],
+        "summary": "Rejects application. Requires shelter role matching with shelterId of application.",
+        "parameters": [
+          {
+            "name": "applicationId",
+            "in": "path",
+            "required": true,
+            "style": "simple",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "400": {
+            "description": "Client error"
+          },
+          "401": {
+            "description": "Client error"
+          }
+        }
+      }
+    },
+    "/applications/{applicationId}/withdraw": {
+      "put": {
+        "tags": [
+          "ApplicationsEndpoints"
+        ],
+        "summary": "Withdraws application. Requires adopter role matching with adopterId of application.",
+        "parameters": [
+          {
+            "name": "applicationId",
+            "in": "path",
+            "required": true,
+            "style": "simple",
+            "schema": {
+              "type": "string"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success"
+          },
+          "400": {
+            "description": "Client error"
+          },
+          "401": {
+            "description": "Client error"
+          }
+        }
+      }
+    }
+  },
+  "components": {
+    "schemas": {
+      "AdopterDTO": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uuid"
+          },
+          "userName": {
+            "type": "string",
+            "nullable": true
+          },
+          "phoneNumber": {
+            "type": "string",
+            "nullable": true
+          },
+          "email": {
+            "type": "string",
+            "nullable": true
+          },
+          "address": {
+            "$ref": "#/components/schemas/NewAddress"
+          },
+          "status": {
+            "$ref": "#/components/schemas/AdopterStatus"
+          }
+        }
+      },
+      "AdopterStatus": {
+        "enum": [
+          0,
+          1,
+          2
+        ],
+        "type": "integer",
+        "format": "int32",
+        "description": "0 - Active, 1 - Blocked, 2 - Deleted"
+      },
+      "NewAddress": {
+        "type": "object",
+        "properties": {
+          "street": {
+            "type": "string",
+            "nullable": true
+          },
+          "city": {
+            "type": "string",
+            "nullable": true
+          },
+          "province": {
+            "type": "string",
+            "nullable": true
+          },
+          "postalCode": {
+            "type": "string",
+            "nullable": true
+          },
+          "country": {
+            "type": "string",
+            "nullable": true
+          }
```

Comment:
Should we have all properties here nullable?

Reply:
I guess not, that's what happen when someone copy-pastes a lil bit too much. Will fix